requirements.txt

```diff
@@ -0,0 +1,3 @@
+selenium
```

Comment:
Lets tie these down to actual versions (i.e. selenuim==1.x.x). We don't want tests breaking unknowinlgy because these packages updated their API.